examples/components/CCFRAUD_VERTICAL_FEDONCE/pretraining/conda.yaml

```diff
@@ -0,0 +1,18 @@
+name: ccfraud_vertical_pretrain_conda_env
+channels:
+  - defaults
+  - pytorch
+  - nvidia
+dependencies:
+  - python=3.8
+  - pip=22.3.1
+  - pytorch=1.13.1
+  - pytorch-cuda=11.6
+  - pip:
+    - azureml-core==1.47.0
```

Comment:
can you reduce dependencies as needed? I believe those are dependencies of the aml_comm helper